tests/models/autoencoders/test_models_vae.py

```diff
@@ -846,6 +846,39 @@ def test_stable_diffusion_encode_sample(self, seed, expected_slice):
         assert torch_all_close(output_slice, expected_output_slice, atol=tolerance)
 
 
+    def test_lora_adapter(self):
+        
+        vae = AutoencoderKL.from_pretrained("stabilityai/sd-turbo", subfolder="vae")
```

Comment:
Ah it's a bit large to test in a PR. Can we leverage something like so?

```python
init_dict, inputs_dict = self.prepare_init_args_and_inputs_for_common()
model = self.model_class(**init_dict)

vae_lora_config = LoraConfig(
            r=16,
            init_lora_weights="gaussian",
            target_modules=target_modules_vae,
        )

vae.add_adapter(vae_lora_config, adapter_name="vae_lora")

active_lora = vae.active_adapters()
self.assertTrue(len(active_lora) == 1)
self.assertTrue(active_lora[0] == "vae_lora")
```

Of course, we would need to update the `target_modules_vae` accordingly as well.

LMK if it's unclear.

Reply:
Thanks @sayakpaul for the code snippet, this helped me figure that I had added the test to the wrong class. It does successfully execute and pass the diffusers tests on my side.